compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -1076,11 +1076,6 @@ impl<'tcx> ParamEnv<'tcx> {
         ty::ParamEnv { packed: CopyTaggedPtr::new(caller_bounds, ParamTag { reveal }) }
     }
 
-    pub fn with_user_facing(mut self) -> Self {
```

Comment:
I also removed this, since it's both (1.) almost certainly wrong to ever use, and (2.) unused.